feugy/melodie
# fix(ui): missing tailwind classes, blocked player and blocked tutorial

### What's in there?
As part of #34 investigations, I found several bugs introduced in beta:

- [x] fix(ui): tailwind kit does not include utility classes
- [x] fix(ui): Player buttons sometimes get stuck
- [x] fix(ui): tutorial is broken since introducing broadcast button

### How to test?

To test the tutorial:
1. drop the db `rm ~/.config/melodie/db.sqlite3`
2. starts melodie again `npm start`

It should work properly until the end.

The tailwind css regression can be seen in atelier: `npm run atelier -w common/ui`. 
Check out broken image tools.
